src/main/flight/alt_hold.c

```diff
@@ -22,118 +22,50 @@
 #include "math.h"
 #include "build/debug.h"
 
+#include "common/maths.h"
 #include "config/config.h"
 #include "fc/runtime_config.h"
-#include "fc/rc.h"
-#include "flight/failsafe.h"
-#include "flight/imu.h"
-#include "flight/pid.h"
-#include "flight/position.h"
+#include "fc/rc.h" // for throttle position
+#include "flight/failsafe.h" // to know if failsafe is active 
+#include "flight/position.h" // to set altitude
```

Comment:
Comments on includes are superfluous
```suggestion
#include "fc/rc.h"
#include "flight/failsafe.h"
#include "flight/position.h"
```

Reply:
They were useful when writing the code, since I knew why they were included.  Then when revising the code it was easy to know if they were still required.

I'll remove them, though.